.github/workflows/publish-snapshot.yml

```diff
@@ -14,22 +16,25 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
-      - name: Install JDK 11
+      - name: Install JDK
         uses: actions/setup-java@v3
         with:
-          distribution: 'zulu'
+          distribution: zulu
           java-version: 11
 
-      - uses: gradle/gradle-build-action@v2
-
-      - name: Retrieve version
+      - name: Set version
         run: |
-          echo "VERSION_NAME=$(cat gradle.properties | grep -w "VERSION_NAME" | cut -d'=' -f2)" >> $GITHUB_ENV
+          echo "ORG_GRADLE_PROJECT_VERSION_NAME=$(git describe --tags --abbrev=0 | awk -F. '/[0-9]+\./{$NF++;print}' OFS=.)-SNAPSHOT" >> $GITHUB_ENV
+
+      - uses: gradle/gradle-build-action@v2
 
-      - name: Publish snapshot
-        run: ./gradlew publishAllPublicationsToMavenCentralRepository
-        if: endsWith(env.VERSION_NAME, '-SNAPSHOT')
```

Comment:
Should not this stay?

Reply:
The `if` was only needed because the release commit would contain a non snapshot version which would then make this workflow fail when it runs on that commit. Since that commit doesn't exist anymore and we are always computing a snapshot version we can just always run it

Reply:
Got it